src/main/scala/cromwell/binding/values/WdlValue.scala

```diff
@@ -44,4 +46,9 @@ trait WdlValue {
   def collectAsSeq[T <: WdlValue](filterFn: PartialFunction[WdlValue, T]): Seq[T] = {
     if (filterFn.isDefinedAt(this)) Seq(filterFn(this)) else Nil
   }
+
+  /**
+   * @return MD5 Hash.
+   */
+  def getHash(implicit hasher: FileHasher): Hash = valueString.md5Sum
```

Comment:
Comment here seems a little unnecessary... perhaps the name of this method can be just `hash`?